agent/config/config.go

```diff
@@ -212,6 +212,8 @@ func EnvironmentConfig() Config {
 	updateDownloadDir := os.Getenv("ECS_UPDATE_DOWNLOAD_DIR")
 	updatesEnabled := utils.ParseBool(os.Getenv("ECS_UPDATES_ENABLED"), false)
 
+	disableMetrics := utils.ParseBool(os.Getenv("ECS_DISABLE_METRICS"), false)
```

Comment:
Can you add this default false value to `DefaultConfig()` above?  (I'd make the comment on that line, but GitHub's diff viewer doesn't seem to want me to.)